src/calc.py

```diff
@@ -9,12 +9,13 @@ def add(a, b, third_operand = 0):
     """
     return a + b + third_operand
 
-def sub(a, b):
+def sub(a, b, third_operand = 0):
     """
     Subtract some numbers
 
     ```py
     sub(8, 3) # 5
+    sub(8, 3, 10) # -5
     ```
     """
-    return a - b
+    return a - b - c
```

Comment:
oh I forgot to update this one too teehee
```suggestion
    return a - b - third_operand
```